packages/ddp-server/livedata_server_tests.js

```diff
@@ -350,6 +350,21 @@ Meteor.methods({
   },
 });
 
+Meteor.publish("livedata_server_test_sub_chain", async function () {
+  await new Promise(r => setTimeout(r, 2000));
+  this.ready();
+  return null;
+});
+
+Tinytest.addAsync("livedata server - waiting for subscription chain", (test, onComplete) => makeTestConnection(test, async (clientConn, serverConn) => {
```

Comment:
I think the format is off in this block. Could you run prettier on it?

If I remember correctly we still have prettier in the root dir.

Reply:
done!